src/index.html

```diff
@@ -5,8 +5,383 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="body__header header">
+      <img
+      src="./images/logo.png"
+      alt="logo"
+      class="logo"
+      >
+      <nav class="nav">
+        <a class="nav__link is-active" href="">APPLE</a>
+        <a class="nav__link" href="">SAMSUNG</a>
+        <a class="nav__link" href="">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+        <a
+        class="nav__link"
+        href=""
+        data-qa="nav-hover"
+        >
+        НОУТБУКИ И КОМПЬЮТЕРЫ
+        </a>
```

Comment:
```suggestion
        <a
          class="nav__link"
          href="#"
          data-qa="nav-hover"
        >
          НОУТБУКИ И КОМПЬЮТЕРЫ
        </a>
```
Отступы, и не оставляй атрибуты пустыми